modules/ossm-removing-multi-cluster-installation-from-development-environment.adoc

```diff
@@ -5,22 +5,22 @@
 [id="ossm-removing-multi-cluster-installation-from-development-environment_{context}"]
 = Removing the multi-cluster topology from a development environment
 
-After experimenting with the multi-cluster functionality in a development environment, remove the multi-cluster topology. 
+After experimenting with the multi-cluster functionality in a development environment, remove the multi-cluster topology from all the clusters. 
 
 .Prerequisites
 
 * You have installed a multi-cluster topology. 
 
 .Procedure
 
-. Remove the cluster1 from a development environment by running the following command:
+. Remove `CLUSTER1` from the development environment by running the following command:
```

Comment:
This doesn't remove the cluster. This removes Istio and the sample applications from the (east) cluster.

Reply:
I updated the two sentences.